mods/tuxemon/db/environment/grass.json

```diff
@@ -2,7 +2,8 @@
   "slug": "grass",
   "battle_graphics": {
     "island_back":"grass_back_island.png",
-    "island_front":"grass_front_island.png"
+    "island_front":"grass_front_island.png",
+    "background":"Background_240px.png"
```

Comment:
we've had issues in the past with mixed case filenames.  some OS will allow mixing them and others won't and it has created issues with loading images.  please rename the file to be all lower case.